pkg/render/typha.go

```diff
@@ -241,9 +241,11 @@ func (c *typhaComponent) typhaRole() *rbacv1.ClusterRole {
 				Resources: []string{
 					"bgpconfigurations",
 					"bgppeers",
+					"bgpfilters",
```

Comment:
It looks like currently `bgpfilters` and `externalnetworks` are only in Enterprise. In that case they should be added in the section after `if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise`